src/tool/cpp/cppwinrt/code_writers.h

```diff
@@ -2,20 +2,28 @@
 
 namespace xlang
 {
-    static void write_license(writer& w)
+    static void write_prolog(writer& w)
```

Comment:
Maybe "preamble" rather than "prolog", not to be confused with https://en.wikipedia.org/wiki/Prolog 😉

Reply:
yes good point